core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -173,7 +173,6 @@ public RouteBuilder regex(Pattern regex) {
      * @see #path(String)
      */
     public RouteBuilder pathWithPrefix(String prefix, String pathPattern) {
-        prefix = ensureAbsolutePath(prefix, "prefix");
```

Comment:
This validation is done in `computePathPrefix` method

Reply:
Without this, a user will get an exception later than when he or she called this method. I think it gives a user better feed back when an exception is triggered as soon as a bad input is given.

Reply:
Added this back.